src/xo-app/jobs/helpers/index.js

```diff
@@ -0,0 +1,115 @@
+import find from 'lodash/find'
+import map from 'lodash/map'
+import React, { Component } from 'react'
+import Select from 'react-select'
+import { propTypes } from 'utils'
+
+@propTypes({
+  autoFocus: propTypes.bool,
+  defaultValue: propTypes.any,
+  disabled: propTypes.bool,
+  multi: propTypes.bool,
+  onChange: propTypes.func,
+  options: propTypes.array,
+  placeholder: propTypes.string,
+  predicate: propTypes.func,
+  required: propTypes.bool
+})
+export class SelectPlainObject extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      value: this._setValue(props.defaultValue, props)
```

Comment:
I agree for the "bad" name.
In fact this is "copied/pasted" from GenericSelect, and I still feel this Component could be a base for the GenericSelect, avoiding code dupplication.